MaybeGame/Player.cs

```diff
@@ -239,12 +240,35 @@ private int SetAttackPower(int strength)
         {
             return 15 + (5 * strength);
         }
+        public int LevelUp(double experiencePoints)
+        {
+            var level = Level;
 
+            if (experiencePoints >= 1)
+            {
+                level++;
+                Console.WriteLine("Level up! Congratulations\n" +
+                    $"Your level is now {level}");
+                ExperiencePoints -= 1;
+            }
+            else
+            {
+                ExperiencePoints += experiencePoints;
+            }
+            
+            return level;
+        }
+        public double SaveExperiencePoints(double experiencePoints)
+        {
+           return SavedExperiencePoints += experiencePoints;
+        }
         public void StatSet(string userName, string clanName)
         {
             UserName = userName;
             ClanName = clanName;
             Race = ChooseARace();
+            ExperiencePoints = SavedExperiencePoints;
+            Level = 1 + LevelUp(ExperiencePoints);
```

Comment:
Avoid magical constants https://en.wikipedia.org/wiki/Magic_number_(programming)

Reply:
Only did this because levels started from 0 not 1.